demosplan/DemosPlanCoreBundle/Logic/FileResponseGenerator/ZipResponseGenerator.php

```diff
@@ -101,25 +106,45 @@ private function fillZipWithData(ZipStream $zipStream, array $file): void
             $this->handleError($e, self::UNKOWN_ERROR, self::XLSX_GENERIC);
         }
 
-        foreach ($file['attachments'] as $attachmentArray) {
-            /** @var File $attachment */
-            foreach ($attachmentArray as $attachment) {
-                try {
+        foreach ($file['attachments'] as $attachmentsArray) {
+            try {
+                foreach ($attachmentsArray['attachments'] as $attachment) {
                     $this->zipExportService->addFileToZipStream(
                         $attachment->getFilePathWithHash(),
                         $file['zipFileName'].'/'.$attachment->getHash().'_'.$attachment->getFilename(),
                         $zipStream
                     );
-                } catch (FileNotFoundException|FileNotReadableException $e) {
-                    $this->handleError($e, self::FIILE_NOT_FOUND_OR_READABLE);
-                    ++$this->errorCount['attachmentNotAddedCount'];
-                } catch (InvalidDataException $e) {
-                    $this->handleError($e, self::FILE_HASH_INVALID);
-                    ++$this->errorCount['attachmentNotAddedCount'];
-                } catch (Exception $e) {
-                    $this->handleError($e, self::UNKOWN_ERROR);
-                    ++$this->errorCount['attachmentUnkownErrorCount'];
                 }
+                /**
+                 * The originalAttachment can be of type { @see File } if there was a stn attachment already set within
+                 * the exportet entity. Or it is an array in case the { @see AssessmentTablePdfExporter } got invoked
+                 * to generate a pdf of the original-stn.
+                 */
+                $originalAttachment = $attachmentsArray['originalAttachment'];
+                if ($originalAttachment instanceof File) {
+                    $this->zipExportService->addFileToZipStream(
+                        $originalAttachment->getFilePathWithHash(),
+                        $file['zipFileName'].'/'.$originalAttachment->getHash().'_'.$originalAttachment->getFilename(),
+                        $zipStream
+                    );
+                }
+                if (is_array($originalAttachment)) {
+                    $zipStream->addFile(
+                        Utf8::toAscii(
```

Comment:
Library Utf8 is deprecated. Use symfony String component here

Reply:
done in [0da0131](https://github.com/demos-europe/demosplan-core/pull/2444/commits/0da0131d53f7a23a10bc0e06ed2406cb32058a0f)